app/src/main/java/org/oppia/android/app/topic/TopicFragment.kt

```diff
@@ -27,13 +27,15 @@ class TopicFragment : InjectableFragment() {
     val internalProfileId = arguments?.getInt(PROFILE_ID_ARGUMENT_KEY) ?: -1
     val topicId = arguments?.getString(TOPIC_ID_ARGUMENT_KEY) ?: TEST_TOPIC_ID_0
     val storyId = arguments?.getString(STORY_ID_ARGUMENT_KEY) ?: ""
+    val isConfigChanged = savedInstanceState != null
 
     return topicFragmentPresenter.handleCreateView(
       inflater,
       container,
       internalProfileId,
       topicId,
-      storyId
+      storyId,
+      isConfigChanged
```

Comment:
Optional, or: ``isConfigChanged = savedInstanceState != null`` which is just as readable without needing a variable.

Reply:
done